### DISCUSSION TITLE
Update about 0_repository_setup.md

### DISCUSSION THREAD

**Discussion by Yevheniia-Rudenko** [MIT-Emerging-Talent/ET6-foundations-group-12#1] [General]
Hi everyone! [Evan mentioned](https://mitemergingtalent.slack.com/archives/C081EFFKCN5/p1734789590494719) that there’s no need to fork our repo; instead, we can already work on this one using branches, etc.

As part of the Setup and Share a Repository task, I’ve already completed the "Under settings in your repository select" section and configured the Collaboration Settings. This will allow us to chat (basically what I’m doing now) and will require approvals if someone pushes something to the main branch.

However, point 1 still needs to be completed, and we’ll definitely need to brainstorm on it together.
![Screenshot 2024-12-22 at 08 16 27](https://github.com/user-attachments/assets/72ec187d-afa6-4044-a01e-504efcfdc3ce)

**Comment by Allukmon** [comment]
## Good job, Yevheniia.

I am _finding my feet_ around this space. Basically, I am just **testing out** what you are reading. 
_**I found out that this comment is markdown supported, Nice**_

**Comment by Awaab98** [comment]
Hello @Yevheniia-Rudenko Where ccan I find the file that you are refering to in the image above.

**Reply by abeddost** [reply]
Hey @Awaab98 Here is the link for it: https://github.com/MIT-Emerging-Talent/ET6-foundations-group-12/blob/main/collaboration/guide/0_repository_setup.md

**Comment by Simi-Solola** [comment]
Noted, thanks

**Comment by Ridwanayinde** [comment]
Noted, thanks. Although, I am still trying to understand GitHub the more.
